### PR TITLE
Fix many issues detected by codespell

### PR DESCRIPTION
**Pull request by mjtrangoni**
I fixed many of the issues detected by [codespell](https://github.com/lucasdemarchi/codespell).

### CODE REVIEW

File: plugins/check_procs.c [hunk 1]
```diff
@@ -289,7 +289,7 @@ main (int argc, char **argv)
                           }
 			}
 
-			/* filter kernel threads (childs of KTHREAD_PARENT)*/
```
Comment (box293) [inline plugins/check_procs.c]: I think this line should remain as is

Comment (mjtrangoni) [reply plugins/check_procs.c]: @box293 good catch!! Fixed. Thanks!

**Comment by mjtrangoni** [comment]
I replaced the `IFF` with `if`, which is wrong. I will be fixing this today.

**Comment by mjtrangoni** [comment]
Done. I revert all the wrong `IFF` replacements.

**Comment by hedenface** [comment]
There was a bit of an issue with a recent merge in check_icmp, but doing a side by side analysis I couldn't find it. If you find it, please open a new PR.

As always, thank you :)

**Pull request merged**